dev/MRTCore/mrt/Microsoft.ApplicationModel.Resources/src/Helper.cpp

```diff
@@ -46,16 +62,25 @@ HRESULT GetDefaultPriFileForCurrentPackage(winrt::hstring& filePath)
 
     std::unique_ptr<wchar_t, decltype(&LocalFree)> resourceFilePtr(resourceFile, LocalFree);
 
+    hr = CheckFile(resourceFile);
+    if (FAILED(hr))
+    {
+        return hr;
+    }
+
     filePath = resourceFile;
 
     return S_OK;
 }
 
 HRESULT GetDefaultPriFile(winrt::hstring& filePath)
 {
-    if (FAILED(GetDefaultPriFileForCurrentPackage(filePath)))
+    HRESULT hr = GetDefaultPriFileForCurrentPackage(filePath);
```

Comment:
>GetDefaultPriFileForCurrentPackage [](http://example.com/codeflow?start=17&length=34)

If the app is packaged, will this function try to get the path of resources.pri? Adding that as a comment would be helpful. From the name "ResourceManager_GetDefaultResourcePathForPackageFullName", I can't tell.

Also, it would be good to have a comment saying why we fallback even in the packaged case. I mean, the name of the first function implies it handles the packaged case, yet we call the second function sometimes with `isPackaged = true`.